stories/accordion.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line
+import StateDecorator from './utils/state-decorator.js';
+import { Accordion, Section } from '../src/accordion';
+
+storiesOf('Accordion', module)
+  .addDecorator(storyFn => <StateDecorator storyFn={storyFn} />)
```

Comment:
StateDecorator is my own invention MWUHAHAHAHAHA

I figured that we want to be able to have interactive examples, without having to create wrapping components for examples. This has the benefit of guaranteeing that users viewing the source/usage tab (not implemented yet) will see a good example of how to use the component, and not something like `<AccordionExample />`.